internal/printer/printer.go

```diff
@@ -52,31 +52,30 @@ var BACKTICK = "`"
 var styleModuleSpecExp = regexp.MustCompile(`(\.css|\.pcss|\.postcss|\.sass|\.scss|\.styl|\.stylus|\.less)$`)
 
 func (p *printer) print(text string) {
-	p.output = append(p.output, text...)
-}
-
-func (p *printer) printRune(c rune) {
-	p.output = append(p.output, byte(c))
+	p.output = append(p.output, []byte(text)...)
```

Comment:
Converting to `[]byte(text)` first will handle `rune` properly (which is a utf8 string)

Reply:
Basically `text...` will append `rune`, but we always need `byte` here.